python/tests/nexus_helpers.py

```diff
@@ -325,6 +336,7 @@ def __init__(self):
         self._detectors: List[Detector] = []
         self._logs: List[Log] = []
         self._instrument_name: Optional[str] = None
+        self._choppers: Optional[Chopper] = []
```

Comment:
Should this be 
```suggestion
        self._choppers: Optional[List[Chopper]] = []
```
? otherwise not sure the default makes sense.